src/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php

```diff
@@ -251,12 +256,45 @@ public function getBreadcrumb(): array
             //TODO needed to fix for add screens
             $breadcrumb['level'] = $menuItem->getLevel() == 3 ? $menuItem->getParent()->getMenuTitle() : null;
             if (!is_null($breadcrumb['level'])) {
-                $breadcrumb['level'] =$this->getI18NHelper()->transBySource($breadcrumb['level']);
+                $breadcrumb['level'] = $this->getI18NHelper()->transBySource($breadcrumb['level']);
             }
         }
         if ($breadcrumb['moduleName']) {
-            $breadcrumb['moduleName'] =$this->getI18NHelper()->transBySource($breadcrumb['moduleName']);
+            $breadcrumb['moduleName'] = $this->getI18NHelper()->transBySource($breadcrumb['moduleName']);
         }
         return $breadcrumb;
     }
+
+    /**
+     * @return array
+     */
+    private function getDateFormat(): array
+    {
+        $dateFormat = $this->getConfigDao()->getValue('admin.localization.default_date_format');
+        switch ($dateFormat) {
+            case 'd-m-Y':
+                $label = 'dd-mm-yyyy';
+                break;
+            case 'm-d-Y':
+                $label = 'mm-dd-yyyy';
+                break;
+            default:
+                $label = 'yyyy-mm-dd';
```

Comment:
shall we maintain a map (assoc array), rather than large switch-case block

Reply:
Yes, replaced with a map